src/vs/workbench/parts/terminal/electron-browser/terminal.ts

```diff
@@ -0,0 +1,18 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import * as os from 'os';
+import platform = require('vs/base/common/platform');
+import processes = require('vs/base/node/processes');
+
+const powerShellExePath =
+		!process.env.hasOwnProperty('PROCESSOR_ARCHITEW6432')
```

Comment:
Actually, now that I'm playing around with it, `process.arch` is going to give you 32-bit.  I think that code that I stole from @daviwil is probably the right way to do it.